README.md

```diff
@@ -62,6 +62,7 @@ A curated list of tools and resources for Platform Engineering.
 - [Terraform Guardrails with OPA](https://compellingcloud.substack.com/p/navigating-safety-a-beginners-guide)
 
 ## Tooling— Kubernetes, PAAS and Cloud services
+- [Argonaut - Deploy apps and infrastructure on your cloud in minutes](https://www.argonaut.dev/)
```

Comment:
Can you please remove "in minutes" part to be similar to the https://www.crunchbase.com/organization/argonaut-862d description,
and I will merge

Reply:
Sure!